routes
raiseError
	<get: '/raiseError'>
	self error: 'This is my error'